conandata.yml

```diff
@@ -1,7 +1,7 @@
 version: "5.7.0-alpha.0"
 requirements:
   - "uranium/(latest)@ultimaker/testing"
-  - "curaengine/(latest)@ultimaker/testing"
+  - "curaengine/(latest)@ultimaker/cura_11482"
```

Comment:
After component test should I make this to testing again? to merge in main?

Reply:
yes